rest/taskana-rest-spring/src/test/java/pro/taskana/TestConfiguration.java

```diff
@@ -0,0 +1,28 @@
+package pro.taskana;
+
+import javax.sql.DataSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.DependsOn;
+import org.springframework.jdbc.datasource.DataSourceTransactionManager;
+import org.springframework.transaction.PlatformTransactionManager;
+
+import pro.taskana.sampledata.SampleDataGenerator;
+
+@SpringBootApplication
+@DependsOn("getTaskanaEngine") // wait for schema to be created BEFORE inserting test data
+public class TestConfiguration {
+
+  @Autowired
```

Comment:
Just fyi: As of Spring 4.3, you no longer need to specify an explicit injection annotation in a single-constructor scenario. (meaning the @Autowired could be deleted here)